io_scene_godot/export_godot.py

```diff
@@ -222,6 +222,137 @@ def export_scene(self):
         if in_edit_mode:
             bpy.ops.object.editmode_toggle()
 
+    def export_collection_scene(self, collection):
+        # pylint: disable-msg=too-many-branches
+        """Decide what objects to export, and export them!"""
+        logging.info("Exporting collection: %s of scene %s",
+                     collection.name, self.scene.name)
+
+        in_edit_mode = False
```

Comment:
I wonder if this should be outside this function? ie in the root "export" function. It's used by all the `export_<thingo>` functions

Reply:
I thought about that, but each if the meaning should be the same I would need to go in each loop in the export function therefore probably changing not a lot